content/docs/fedora-releases-guide.md

```diff
@@ -0,0 +1,138 @@
+---
+title: How to do Fedora releases with Packit
+weight: 9
+---
+# How to do Fedora releases with Packit
+
+Let's split the release process into single steps:
+1. New upstream release 
+2. Uploading archive to lookaside cache
+3. Updating dist-git content 
+4. Koji builds
+5. Bodhi updates
+
+Doing Fedora releases with Packit means utilising our 3 jobs - `propose_downstream`, `koji_build` and 
+`bodhi_update` - where each of the jobs takes care of a different part of the release process.
+
+## Propose downstream job
+For enabling the propose downstream job, you need to have [Packit Service installed](/docs/packit-service) 
```

Comment:
The use of `propose downstream` is a bit inconsistent in this part of the text, it is used both with and without the underscore. Perhaps we should just use `propose_downstream` everywhere? Although this may look a bit weird in the text.

Reply:
I am really not sure, using `propose_downstream` everywhere looks weird to me, I know it isn't 100% consistent, but I tried to use `propose_downstream` when referencing directly the configuration

Reply:
Agreed, lets keep `propose_downstream` for when referencing the config specifically